src/ptvsd/adapter/__main__.py

```diff
@@ -5,20 +5,28 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 import sys
+import argparse
 
 # WARNING: ptvsd and submodules must not be imported on top level in this module,
 # and should be imported locally inside main() instead.
 
 
+def _get_parsed_args():
+    parser = argparse.ArgumentParser(prog="ptvsd.adapter")
+    parser.add_argument("--debug-port", nargs="?", default=None, const=6789)
+    return parser.parse_args(sys.argv[1:])
```

Comment:
I think you can skip `prog=` and omit the argument here - it should take the default `prog=sys.argv[0]` then. So if you do `python -m ptvsd.adapter`, it'll do the same thing.